docs/advanced/DevWorkflow.md

```diff
@@ -0,0 +1,26 @@
+---
+id: DevWorkflow
```

Comment:
You need to add the page in the menu system here < https://github.com/net-daemon/docs/blob/main/sidebars.js#L34>